environments/key4hep-common/packages.yaml

```diff
@@ -60,6 +60,9 @@ packages:
     variants: ~openmp
   gperftools:
     variants: ~libunwind
+  hdf5:
+    variants: +cxx+hl
+    require: "+cxx+hl"
```

Comment:
Isn't the `require` redundant? It doesn't need to be specified when variants is specified.

Reply:
No, the concretiser may override this unless it's under spec or require like this

Reply:
Maybe we could switch all the `variants` to `require`? I would have to check when that became available.

Reply:
> Maybe we could switch all the `variants` to `require`? I would have to check when that became available.

Sounds good